src/patternfly/components/Nav/nav.scss

```diff
@@ -135,9 +135,36 @@
     flex-grow: 1;
   }
 
+  // stylelint-disable max-nesting-depth
   .#{$menu} {
     --#{$menu}--MinWidth: 100%;
+
+    &.pf-m-drilldown {
+      --#{$menu}--PaddingBlockStart: 0;
+      --#{$menu}--PaddingBlockEnd: 0;
+      --#{$menu}--BackgroundColor: var(--#{$nav}--BackgroundColor);
+      --#{$menu}--BoxShadow: none;
+      --#{$menu}__list-item--Color: var(--#{$nav}__link--Color);
+      --#{$menu}__list-item--BackgroundColor: transparent;
+      --#{$menu}__list-item--hover--BackgroundColor: var(--#{$nav}__link--hover--BackgroundColor);
+      --#{$menu}__list-item--hover--BackgroundColor: transparent;
+      --#{$menu}__item--BackgroundColor: var(--#{$nav}__link--BackgroundColor);
+
+      .#{$menu}__list {
+        gap: var(--#{$nav}__subnav--RowGap);
+      }
+    
+      .#{$menu}__item {
+        border-radius: var(--#{$nav}__link--BorderRadius);
+    
+        &:hover,
+        &:focus {
+          --#{$menu}__item--BackgroundColor: var(--#{$nav}__link--hover--BackgroundColor);
+        }
```

Comment:
Curious if we could remove this and get the same with `--pf-v6-c-menu__list-item--hover--BackgroundColor: var(--pf-v6-c-nav__link--hover--BackgroundColor);` in the `&.pf-m-drilldown` above? We would then want to update the `border-radius` so it goes on `.pf-v6-c-menu__list-item` and `.pf-v6-c-menu__list-item::before` (or maybe just `.pf-v6-c-menu__list-item::before` if that works?) and clean up the existing item/list-item background vars on `&.pf-m-drilldown`